pages/advocates.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <main>
+  <div>
```

Comment:
Same as before, could you double check if this fits better as `article` than a `div`?

Reply:
Updated in https://github.com/Qiskit/qiskit.org/pull/3441/commits/8813034834a0b3ef57c06bdf5a74df881100161f